### PR TITLE
DintegratorCables

### PR DESCRIPTION
**Pull request by KhWahba**
This is the 2 2d (x,y) double integrators, each connected to a mass "m0" with two cables.

### CODE REVIEW

File: src/DintegratorCables.cpp [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+  std::cout << "Parameters" << std::endl;
+  this->params.write(std::cout);
+  std::cout << "***" << std::endl;
+
+  name = "DintegratorCables";
+  translation_invariance = 2;
+  invariance_reuse_col_shape = false;
+  nx_col = 8; 
+  nx_pr = 2;
+  is_2d = false;
```
Comment (whoenig) [inline src/DintegratorCables.cpp]: it is a 2D problem?

File: src/robot_models.cpp [hunk 2]
```diff
@@ -73,8 +74,8 @@ std::unique_ptr<Model_robot> robot_factory(const char *file,
   } else if (dynamics == "quad3dpayload_point") {
     return std::make_unique<Model_quad3dpayload_n>(file, p_lb, p_ub);
   }
-  else if (dynamics == "quad3dpayload_n") {
-    return std::make_unique<Model_quad3dpayload_n>(file, p_lb, p_ub);
+  else if (dynamics == "DintegratorCables") {
+    return std::make_unique<DintegratorCables>(file, p_lb, p_ub);
```
Comment (whoenig) [inline src/robot_models.cpp]: shouldn't that be an addition, not a replacement of quad3dpayload_n?

File: test/test_DintegratorCables.cpp [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+#include "dynobench/DintegratorCables.hpp"
+
+#define base_path "../"
+
+
+using namespace dynobench;
+
+BOOST_AUTO_TEST_CASE(t_DintegratorCables) {
+
+  std::cout << "Hello Khaled" << std::endl;
```
Comment (whoenig) [inline test/test_DintegratorCables.cpp]: :-) ?

File: test/test_DintegratorCables.cpp [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+  dynobench::DintegratorCables_params params;
+  params.read_from_yaml(base_path "models/DintegratorCables.yaml");
+  auto model = mk<dynobench::DintegratorCables>(params);
+
+}
+
+BOOST_AUTO_TEST_CASE(t_DintegratorCables_dynamics) { 
+
+  std::cout << "Hello: \n" << std::endl;
```
Comment (whoenig) [inline test/test_DintegratorCables.cpp]: ?

**Review by whoenig — APPROVED** [review]
Looks good - small comments see review.

File: src/DintegratorCables.cpp (hunk 1 above)
Comment (KhWahba) [reply src/DintegratorCables.cpp]: yes, will fix that.

File: src/robot_models.cpp (hunk 2 above)
Comment (KhWahba) [reply src/robot_models.cpp]: You are right, it was a mistake.

File: test/test_DintegratorCables.cpp (hunk 3 above)
Comment (KhWahba) [reply test/test_DintegratorCables.cpp]: Trying to cheer myself up :D 
will remove it.

**Pull request merged**